vendor/bat-native-ads/src/bat/ads/internal/geographic/subdivision/subdivision_targeting.cc

```diff
@@ -84,7 +85,8 @@ void SubdivisionTargeting::OnSubdivisionTargetingCodePrefChanged() {
   subdivision_code_ = AdsClientHelper::GetInstance()->GetStringPref(
       prefs::kSubdivisionTargetingCode);
 
-  MaybeFetch();
+  const std::string locale = LocaleManager::GetInstance()->GetLocale();
+  MaybeFetchForLocale(locale);
```

Comment:
Maybe cleaner to revert this back to `MaybeFetch()` and then decouple `MaybeAllowForLocale` from `MaybeFetch`. We can then add a new public function `MaybeAllow`